### ISSUE TITLE
Import.rake username,table.csv

### ISSUE THREAD

**Issue by jesusvazquez** [CartoDB/cartodb#4768]
Hi guys,

I was trying to import a dataset in csv format using the import rake script in /lib/tasks

My local user is cdb and my csv is named table.csv. I execute this command:
```bundle exec rake cartodb:import['cdb','/tmp/table.csv']  --trace```

And I get this error

```
** Invoke cartodb:import (first_time)
** Invoke environment (first_time)
** Execute environment
** Execute cartodb:import

  [1m[36mCarto::UserTable Load (1.3ms)[0m  [1mSELECT "user_tables".* FROM "user_tables" WHERE "user_tables"."name" = 'table' AND "user_tables"."user_id" = '0591b0e1-6109-43ad-8b54-dce7f6302f1b' LIMIT 1[0m
=====================
PG::Error: ERROR:  syntax error at or near "table"
LINE 1: SELECT count(*) as count from table
                                      ^
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:145:in `async_exec'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:145:in `block in execute_query'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/logging.rb:33:in `log_yield'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:145:in `execute_query'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:132:in `block in execute'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:111:in `check_disconnect_errors'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:132:in `execute'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:411:in `_execute'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:242:in `block (2 levels) in execute'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:423:in `check_database_errors'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:242:in `block in execute'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/connecting.rb:236:in `block in synchronize'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/connection_pool/threaded.rb:104:in `hold'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/connecting.rb:236:in `synchronize'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:242:in `execute'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/dataset/actions.rb:778:in `execute'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:523:in `fetch_rows'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/dataset/actions.rb:154:in `each'
/var/lib/gems/1.9.1/gems/sequel_pg-1.6.3/lib/sequel_pg/sequel_pg.rb:58:in `each'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/dataset/actions.rb:546:in `single_record'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/dataset/actions.rb:209:in `first'
/home/ubuntu/www/production.cartodb.com/current/lib/importer/lib/cartodb-migrator/migrator.rb:122:in `migrate!'
/home/ubuntu/www/production.cartodb.com/current/app/models/table.rb:378:in `import_to_cartodb'
/home/ubuntu/www/production.cartodb.com/current/app/models/table.rb:461:in `before_create'
/home/ubuntu/www/production.cartodb.com/current/app/models/table/user_table.rb:184:in `before_create'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1580:in `block (2 levels) in _save'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:867:in `around_create'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1578:in `block in _save'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:867:in `around_save'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1573:in `_save'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1254:in `block (2 levels) in save'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1722:in `block in checked_transaction'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/query.rb:323:in `_transaction'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/query.rb:285:in `block in transaction'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/connecting.rb:236:in `block in synchronize'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/connection_pool/threaded.rb:104:in `hold'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/connecting.rb:236:in `synchronize'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/query.rb:278:in `transaction'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1722:in `checked_transaction'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1254:in `block in save'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1710:in `checked_save_failure'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1254:in `save'
/home/ubuntu/www/production.cartodb.com/current/app/models/table.rb:81:in `save'
/home/ubuntu/www/production.cartodb.com/current/app/models/table_registrar.rb:18:in `register'
/home/ubuntu/www/production.cartodb.com/current/app/connectors/importer.rb:174:in `persist_metadata'
/home/ubuntu/www/production.cartodb.com/current/app/connectors/importer.rb:64:in `register'
/home/ubuntu/www/production.cartodb.com/current/app/connectors/importer.rb:49:in `block in run'
/home/ubuntu/www/production.cartodb.com/current/app/connectors/importer.rb:48:in `each'
/home/ubuntu/www/production.cartodb.com/current/app/connectors/importer.rb:48:in `run'
/home/ubuntu/www/production.cartodb.com/current/app/models/data_import.rb:617:in `new_importer'
/home/ubuntu/www/production.cartodb.com/current/app/models/data_import.rb:350:in `dispatch'
/home/ubuntu/www/production.cartodb.com/current/app/models/data_import.rb:161:in `run_import!'
/home/ubuntu/www/production.cartodb.com/current/lib/tasks/import.rake:21:in `block (2 levels) in <top (required)>'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:205:in `call'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:205:in `block in execute'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:200:in `each'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:200:in `execute'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:158:in `block in invoke_with_call_chain'
/usr/lib/ruby/1.9.1/monitor.rb:211:in `mon_synchronize'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:151:in `invoke_with_call_chain'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:144:in `invoke'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:116:in `invoke_task'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:94:in `block (2 levels) in top_level'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:94:in `each'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:94:in `block in top_level'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:133:in `standard_exception_handling'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:88:in `top_level'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:66:in `block in run'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:133:in `standard_exception_handling'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:63:in `run'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/bin/rake:33:in `<top (required)>'
/usr/local/bin/rake:23:in `load'
/usr/local/bin/rake:23:in `<main>'
=====================
PG::Error: ERROR:  syntax error at or near "table"
LINE 1: SELECT count(*) as count from table
                                      ^
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:145:in `async_exec'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:145:in `block in execute_query'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/logging.rb:33:in `log_yield'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:145:in `execute_query'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:132:in `block in execute'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:111:in `check_disconnect_errors'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:132:in `execute'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:411:in `_execute'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:242:in `block (2 levels) in execute'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:423:in `check_database_errors'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:242:in `block in execute'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/connecting.rb:236:in `block in synchronize'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/connection_pool/threaded.rb:104:in `hold'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/connecting.rb:236:in `synchronize'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:242:in `execute'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/dataset/actions.rb:778:in `execute'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:523:in `fetch_rows'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/dataset/actions.rb:154:in `each'
/var/lib/gems/1.9.1/gems/sequel_pg-1.6.3/lib/sequel_pg/sequel_pg.rb:58:in `each'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/dataset/actions.rb:546:in `single_record'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/dataset/actions.rb:209:in `first'
/home/ubuntu/www/production.cartodb.com/current/lib/importer/lib/cartodb-migrator/migrator.rb:122:in `migrate!'
/home/ubuntu/www/production.cartodb.com/current/app/models/table.rb:378:in `import_to_cartodb'
/home/ubuntu/www/production.cartodb.com/current/app/models/table.rb:461:in `before_create'
/home/ubuntu/www/production.cartodb.com/current/app/models/table/user_table.rb:184:in `before_create'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1580:in `block (2 levels) in _save'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:867:in `around_create'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1578:in `block in _save'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:867:in `around_save'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1573:in `_save'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1254:in `block (2 levels) in save'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1722:in `block in checked_transaction'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/query.rb:323:in `_transaction'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/query.rb:285:in `block in transaction'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/connecting.rb:236:in `block in synchronize'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/connection_pool/threaded.rb:104:in `hold'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/connecting.rb:236:in `synchronize'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/query.rb:278:in `transaction'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1722:in `checked_transaction'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1254:in `block in save'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1710:in `checked_save_failure'
/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1254:in `save'
/home/ubuntu/www/production.cartodb.com/current/app/models/table.rb:81:in `save'
/home/ubuntu/www/production.cartodb.com/current/app/models/table_registrar.rb:18:in `register'
/home/ubuntu/www/production.cartodb.com/current/app/connectors/importer.rb:174:in `persist_metadata'
/home/ubuntu/www/production.cartodb.com/current/app/connectors/importer.rb:64:in `register'
/home/ubuntu/www/production.cartodb.com/current/app/connectors/importer.rb:49:in `block in run'
/home/ubuntu/www/production.cartodb.com/current/app/connectors/importer.rb:48:in `each'
/home/ubuntu/www/production.cartodb.com/current/app/connectors/importer.rb:48:in `run'
/home/ubuntu/www/production.cartodb.com/current/app/models/data_import.rb:617:in `new_importer'
/home/ubuntu/www/production.cartodb.com/current/app/models/data_import.rb:350:in `dispatch'
/home/ubuntu/www/production.cartodb.com/current/app/models/data_import.rb:161:in `run_import!'
/home/ubuntu/www/production.cartodb.com/current/lib/tasks/import.rake:21:in `block (2 levels) in <top (required)>'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:205:in `call'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:205:in `block in execute'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:200:in `each'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:200:in `execute'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:158:in `block in invoke_with_call_chain'
/usr/lib/ruby/1.9.1/monitor.rb:211:in `mon_synchronize'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:151:in `invoke_with_call_chain'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:144:in `invoke'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:116:in `invoke_task'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:94:in `block (2 levels) in top_level'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:94:in `each'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:94:in `block in top_level'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:133:in `standard_exception_handling'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:88:in `top_level'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:66:in `block in run'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:133:in `standard_exception_handling'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:63:in `run'
/var/lib/gems/1.9.1/gems/rake-0.9.2.2/bin/rake:33:in `<top (required)>'
/usr/local/bin/rake:23:in `load'
/usr/local/bin/rake:23:in `<main>'
2015-07-29 11:00:41 UTC: Running on server cdb with PID: 25306
2015-07-29 11:00:41 UTC: new_importer()
2015-07-29 11:00:41 UTC: Fetching datasource public_url metadata for item id /tmp/table.csv
2015-07-29 11:00:41 UTC: File will be downloaded from /tmp/table.csv
2015-07-29 11:00:41 UTC: Before importer run
2015-07-29 11:00:41 UTC: Starting import for /tmp/table.csv
2015-07-29 11:00:41 UTC: Unpacking /tmp/table.csv
2015-07-29 11:00:41 UTC: Filename: /tmp/imports/20150729-25306-1i0fgin/table.csv Size (bytes): 341
2015-07-29 11:00:41 UTC: Importing data from /tmp/imports/20150729-25306-1i0fgin/table.csv
2015-07-29 11:00:41 UTC: File-based import load
2015-07-29 11:00:41 UTC: Detected encoding ISO-8859-1
2015-07-29 11:00:41 UTC: Using database connection with {:adapter=>"postgres", :encoding=>"unicode", :host=>"localhost", :port=>5432, :database=>"cartodb_dev_user_0591b0e1-6109-43ad-8b54-dce7f6302f1b_db", :username=>"postgres", :conn_validator_timeout=>900, :user=>"development_cart
2015-07-29 11:00:42 UTC: ogr2ogr call:            OSM_USE_CUSTOM_INDEXING=NO PG_USE_COPY=YES PGCLIENTENCODING=ISO-8859-1 /usr/bin/ogr2ogr -f PostgreSQL  PG:"host=localhost port=5432 user=development_cartodb_user_0591b0e1-6109-43ad-8b54-dce7f6302f1b dbname=cartodb_dev_user_0591b0e1-
2015-07-29 11:00:42 UTC: ogr2ogr output:          
2015-07-29 11:00:42 UTC: ogr2ogr exit code:       0
2015-07-29 11:00:42 UTC: Georeferencing...
2015-07-29 11:00:42 UTC: Disabling autovacuum for "cdb_importer"."importer_0d9f519035e111e5a9d0080027880ca6"
2015-07-29 11:00:42 UTC: Creating the_geom from the_geom column
2015-07-29 11:00:42 UTC: replace empty strings by nulls?
2015-07-29 11:00:42 UTC: string column found, replacing
2015-07-29 11:00:42 UTC: Running batched query by id in "cdb_importer"."importer_0d9f519035e111e5a9d0080027880ca6": 
                  UPDATE "cdb_importer"."importer_0d9f519035e111e5a9d0080027880ca6"
                  SET the_geom=NULL
                  WHERE the_geom=''
         
2015-07-29 11:00:42 UTC: Using as id 'cartodb_processed_4145269516095673910'
2015-07-29 11:00:42 UTC: Finished batched query by 'cartodb_processed_4145269516095673910' in "cdb_importer"."importer_0d9f519035e111e5a9d0080027880ca6": query
2015-07-29 11:00:42 UTC: geometrifying
2015-07-29 11:00:42 UTC: casting the_geom to geometry
2015-07-29 11:00:42 UTC: Converting to 2D point
2015-07-29 11:00:42 UTC: Running batched query by id in "cdb_importer"."importer_0d9f519035e111e5a9d0080027880ca6": 
                UPDATE "cdb_importer"."importer_0d9f519035e111e5a9d0080027880ca6"
                SET the_geom = public.ST_Force_2D(the_geom)
              
2015-07-29 11:00:42 UTC: Using as id 'cartodb_processed_4145269516095673910'
2015-07-29 11:00:42 UTC: Finished batched query by 'cartodb_processed_4145269516095673910' in "cdb_importer"."importer_0d9f519035e111e5a9d0080027880ca6": query
2015-07-29 11:00:42 UTC: geometrified
2015-07-29 11:00:42 UTC: Enabling autovacuum for "cdb_importer"."importer_0d9f519035e111e5a9d0080027880ca6"
2015-07-29 11:00:42 UTC: Georeferenced
2015-07-29 11:00:42 UTC: Finished importing data from /tmp/imports/20150729-25306-1i0fgin/table.csv
2015-07-29 11:00:42 UTC: Proceeding to register
2015-07-29 11:00:42 UTC: Before renaming from importer_0d9f519035e111e5a9d0080027880ca6 to table
2015-07-29 11:00:42 UTC: Before moving schema 'table' from cdb_importer to public
2015-07-29 11:00:42 UTC: Before persisting metadata 'table' data_import_id: 88a1d9cc-c938-4f20-afe2-ab46a8625134
2015-07-29 11:00:42 UTC: Exception: PG::Error: ERROR:  syntax error at or near "table"
LINE 1: SELECT count(*) as count from table
                                      ^

2015-07-29 11:00:42 UTC: ["/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:145:in `async_exec'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:145:in `block in execute_query'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/logging.rb:33:in `log_yield'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:145:in `execute_query'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:132:in `block in execute'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:111:in `check_disconnect_errors'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:132:in `execute'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:411:in `_execute'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:242:in `block (2 levels) in execute'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:423:in `check_database_errors'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:242:in `block in execute'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/connecting.rb:236:in `block in synchronize'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/connection_pool/threaded.rb:104:in `hold'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/connecting.rb:236:in `synchronize'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:242:in `execute'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/dataset/actions.rb:778:in `execute'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/adapters/postgres.rb:523:in `fetch_rows'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/dataset/actions.rb:154:in `each'", "/var/lib/gems/1.9.1/gems/sequel_pg-1.6.3/lib/sequel_pg/sequel_pg.rb:58:in `each'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/dataset/actions.rb:546:in `single_record'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/dataset/actions.rb:209:in `first'", "/home/ubuntu/www/production.cartodb.com/current/lib/importer/lib/cartodb-migrator/migrator.rb:122:in `migrate!'", "/home/ubuntu/www/production.cartodb.com/current/app/models/table.rb:378:in `import_to_cartodb'", "/home/ubuntu/www/production.cartodb.com/current/app/models/table.rb:461:in `before_create'", "/home/ubuntu/www/production.cartodb.com/current/app/models/table/user_table.rb:184:in `before_create'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1580:in `block (2 levels) in _save'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:867:in `around_create'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1578:in `block in _save'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:867:in `around_save'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1573:in `_save'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1254:in `block (2 levels) in save'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1722:in `block in checked_transaction'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/query.rb:323:in `_transaction'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/query.rb:285:in `block in transaction'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/connecting.rb:236:in `block in synchronize'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/connection_pool/threaded.rb:104:in `hold'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/connecting.rb:236:in `synchronize'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/database/query.rb:278:in `transaction'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1722:in `checked_transaction'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1254:in `block in save'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1710:in `checked_save_failure'", "/var/lib/gems/1.9.1/gems/sequel-3.42.0/lib/sequel/model/base.rb:1254:in `save'", "/home/ubuntu/www/production.cartodb.com/current/app/models/table.rb:81:in `save'", "/home/ubuntu/www/production.cartodb.com/current/app/models/table_registrar.rb:18:in `register'", "/home/ubuntu/www/production.cartodb.com/current/app/connectors/importer.rb:174:in `persist_metadata'", "/home/ubuntu/www/production.cartodb.com/current/app/connectors/importer.rb:64:in `register'", "/home/ubuntu/www/production.cartodb.com/current/app/connectors/importer.rb:49:in `block in run'", "/home/ubuntu/www/production.cartodb.com/current/app/connectors/importer.rb:48:in `each'", "/home/ubuntu/www/production.cartodb.com/current/app/connectors/importer.rb:48:in `run'", "/home/ubuntu/www/production.cartodb.com/current/app/models/data_import.rb:617:in `new_importer'", "/home/ubuntu/www/production.cartodb.com/current/app/models/data_import.rb:350:in `dispatch'", "/home/ubuntu/www/production.cartodb.com/current/app/models/data_import.rb:161:in `run_import!'", "/home/ubuntu/www/production.cartodb.com/current/lib/tasks/import.rake:21:in `block (2 levels) in <top (required)>'", "/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:205:in `call'", "/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:205:in `block in execute'", "/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:200:in `each'", "/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:200:in `execute'", "/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:158:in `block in invoke_with_call_chain'", "/usr/lib/ruby/1.9.1/monitor.rb:211:in `mon_synchronize'", "/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:151:in `invoke_with_call_chain'", "/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/task.rb:144:in `invoke'", "/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:116:in `invoke_task'", "/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:94:in `block (2 levels) in top_level'", "/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:94:in `each'", "/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:94:in `block in top_level'", "/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:133:in `standard_exception_handling'", "/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:88:in `top_level'", "/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:66:in `block in run'", "/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:133:in `standard_exception_handling'", "/var/lib/gems/1.9.1/gems/rake-0.9.2.2/lib/rake/application.rb:63:in `run'", "/var/lib/gems/1.9.1/gems/rake-0.9.2.2/bin/rake:33:in `<top (required)>'", "/usr/local/bin/rake:23:in `load'", "/usr/local/bin/rake:23:in `<main>'"]
2015-07-29 11:00:42 UTC: ERROR!

===LOG END===
```

If I rename the table.csv to example.csv everything works fine so this is not a big issue. Just letting you know about this.

**Comment by Kartones** [comment]
Yep, it is a known issue :( There is a list of diallowed names.

But it should try renaming, so thanks, will take a look

**Comment by gfiorav** [comment]
This issue is now fixed in PR #4983

**Comment by gfiorav** [comment]
Forgot to mention: 

The scope of this issue is bigger than a rake task. Users can't import any file with name matching a PostgreSQL special word.
